plugin.video.retrospect/resources/libs/connectivity/dnshttpadapter.py

```diff
@@ -0,0 +1,69 @@
+# ===============================================================================
+# LICENSE Retrospect-Framework - CC BY-NC-ND
+# ===============================================================================
+# This work is licenced under the Creative Commons
+# Attribution-Non-Commercial-No Derivative Works 3.0 Unported License. To view a
+# copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/3.0/
+# or send a letter to Creative Commons, 171 Second Street, Suite 300,
+# San Francisco, California 94105, USA.
+# ===============================================================================
+
+import requests
+from requests.adapters import HTTPAdapter, DEFAULT_POOLSIZE, DEFAULT_RETRIES, DEFAULT_POOLBLOCK
+
```

Comment:
Why do you need to resolve dns entries on your own? This should be handled by the OS itself.